nipype/interfaces/io.py

```diff
@@ -233,9 +298,12 @@ class DataSink(IOBase):
         >>> ds.run()  # doctest: +SKIP
```

Comment:
It would be good to add an S3 example here (even if it would be just with +SKIP).